Marlin/Configuration_adv.h

```diff
@@ -656,6 +656,7 @@
 //#define Z_STEPPER_AUTO_ALIGN
 #if ENABLED(Z_STEPPER_AUTO_ALIGN)
   // Define probe X and Y positions for Z1, Z2 [, Z3]
+  // Optional when using Z_STEPPER_ALIGN_KNOWN_STEPPER_POSITIONS.
```

Comment:
I hadn't rolled this comment into all the examples, since I didn't know if we would want to phrase it this way. This is going to conflict with @InsanityAutomation's change anyway, so maybe we should just skip the comment for now.

Of course maybe this enter commit will be redundant with his changes, and it should just be merged into his branch rather than bf2?